libraries.json

```diff
@@ -433,5 +433,12 @@
     "description": "xUnit.net is a free, open source, community-focused unit testing tool for the .NET Framework.",
     "image": "https://avatars2.githubusercontent.com/u/2092016?s=200&v=4",
     "language": "C#"
+  },
+  "Yii": {
+    "imports": ["yii\\", "yiisoft/yii"],
+    "technologies": ["Web Development", "PHP Frameworks"],
```

Comment:
technologies": ["Web Development", "PHP Frameworks"],

Reply:
Sorry, the code review was done by my test account. But anyway, Yii deserves an "MVC" tag too.